tests/e2e/default/navigation/more-options-menu/offline-user.wdio-spec.js

```diff
@@ -0,0 +1,49 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const utils = require('../../../../utils');
+const seeder = require('./seeder');
+
+describe('Export enabled when there are items: messages, contacts, peope', async () => {
+  before(async () => {
+    await seeder.saveDocs();
```

Comment:
I just came across the experience: 
I didn't read the seed data first; I had to find how reports were created to understand the setup because we are testing reports with content_type XML and not XML. 
It would have helped to know easier the type of reports the seeder included from the start.

Reply:
I like having the shared utility for seeding test data that is going to be common between `more-options-menu` tests. But I do have the following reservations:

- As @latin-panda mentioned, it needs to be flexible enough that we can easily adapt it for new tests.
- If it is generic functionality (not specific to `more-options-menu` behavior) we should put the utility code in a more reusable place so other tests have access.

IMHO the seeder is a good idea, but maybe we don't include the `saveDocs` function at all?  The seeder can export a more fine-grained set of functionality that lets consumers have more control over what is written (and also gives more visibility to what is happening).  My general approach to test utilities is the further away from the actual test you go, the more specific the functionality should be.  It seems reasonable to have a function within a `describe` block (or maybe within a test file) that is responsible for just completely setting up the data for the tests.  But once you move to a separate util file, further from the test code, it is easier to read and maintain more specific purpose-built functions.

Reply:
Thanks Josh! Yes agree; functions like `saveDocs`, `login`, etc., can stay out of the seeder to increase test readability and flexibility. Seeder can focus on building standard data 🤔